editor/src/components/user-bar.tsx

```diff
@@ -102,6 +102,10 @@ const MultiplayerUserBar = React.memo(() => {
     return others.slice(MAX_VISIBLE_OTHER_PLAYERS)
   }, [others])
 
+  const offlineOthers = Object.values(collabs).filter((collab) => {
+    return !others.some((other) => other.id === collab.id) && collab.id !== myUser.id
```

Comment:
```suggestion
    return collab.id !== myUser.id && !others.some((other) => other.id === collab.id)
```
Minor point, but if you re-order this check so that the `collab.id !== myUser.id` check is at the front, whenever that evaluates to `false` it will be able to skip walking the entire `others` array